src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/EnvironmentTests.cs

```diff
@@ -574,6 +574,44 @@ public void GetLogicalDrives_Windows_MatchesExpectedLetters()
             }
         }
 
+        [Fact]
+        public void TestCpuUsage()
+        {
+            if (PlatformDetection.IsiOS && PlatformDetection.IstvOS)
+            {
+                Assert.Throws<PlatformNotSupportedException>(() => Environment.CpuUsage);
+            }
+            else
+            {
+                TimeSpan delta = TimeSpan.FromMinutes(1);
+
+                for (int i = 0; i < 10; i++)
```

Comment:
I do not think we need to run 10 iterations of this loop. One iteration should be enough - if there is an intermittent issue, we will find out even with one iteration.